packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -364,6 +364,7 @@ export function buildRootObject(_powers, _params, testJigSetter = undefined) {
       getBrandForIssuer: issuer => issuerTable.getByIssuer(issuer).brand,
       getIssuerForBrand: brand => issuerTable.getByBrand(brand).issuer,
       getAmountMath,
+      getAdminNode: () => E(zoeInstanceAdmin).getAdminNode(),
```

Comment:
I don't think we want to expose this. Is there a reason we must?

Reply:
I'm not sure about terminate(), but I feel strongly that the owner should have access to done() and adminData().

It's the only way for the owner (or anyone) to get access to these. The fact that it's in zcf means that access will be visible via the contract source, which seems like a strong argument for putting it here.